lib/spinor_noise.cu

```diff
@@ -163,8 +163,12 @@ namespace quda {
       spinorNoise<real,Ns,24>(src, randstates, type);
     } else if (src.Ncolor() == 32) {
       spinorNoise<real,Ns,32>(src, randstates, type);
+    } else if (src.Ncolor() == 64) {
+      spinorNoise<real,Ns,64>(src, randstates, type);
+    } else if (src.Ncolor() == 96) {
+      spinorNoise<real,Ns,96>(src, randstates, type);
     } else {
-      errorQuda("nColor = %d not implemented", src.Ncolor());
+      errorQuda("Ncolor = %d not implemented", src.Ncolor());
```

Comment:
Style.  Camel case is used typically for variables or templates that define a number of something, e.g., `n` amounts of something, e.g., `nSpin`, `nColor`, `nDim`.  Standard capitalization of these quantities saved for function name.  So in this print statement, I prefer camel case.

Reply:
Fixed here, and elsewhere in the file where it's relevant. Not exactly sure why I had that in the first place.